demo/shared.py

```diff
@@ -35,4 +35,13 @@ def demo_page(*components: AnyComponent, title: str | None = None) -> list[AnyCo
                 *components,
             ],
         ),
+        c.Footer(
+            extra_text='FastUI Demo - Extra footer text',
+            links=[
+                c.Link(components=[c.Text(text='Home')], on_click=GoToEvent(url='/')),
+                c.Link(components=[c.Text(text='Components')], on_click=GoToEvent(url='/components')),
```

Comment:
better to have a link to github in the footer, makes more sense than repeating the menu links.

Reply:
Done